engineering_tools/carma

```diff
@@ -252,6 +252,70 @@ carma-config__status() {
     fi
 }
 
+carma__exec() {
+    local TARGET_IMAGE="usdotfhwastol/carma-platform:latest"
```

Comment:
Is there a way to check if the `latest` image exists? Might not be bad to print an error if it's not found and there wasn't anything else specified.

Reply:
Added